### ISSUE TITLE
.fls files are put in output directory, should be placed in aux directory instead

### ISSUE THREAD

**Issue by bersbersbers** [MiKTeX/miktex#634]
- [x] I have read the [MiKTeX Contributing Guidelines](https://github.com/MiKTeX/miktex/blob/master/CONTRIBUTING.md)

In my opinion, `.fls` files (related to `latex -recorder`, see https://linux.die.net/man/1/pdflatex) should not be put in output directory, but in aux directory.

**Comment by edocevoli** [comment]
This has been fixed with MiKTeX 20.10: `.fls` are now installed in the directory for auxiliary files (`--aux-directory=DIR`).

**Comment by K4rl-M4rx** [comment]
Maybe more info can help?

I put above MWE(test.tex) in ~/test/ folder. And I do this 

>  latexmk -auxdir=./temp -outdir=./out test.tex

Then in the ~/test/out/ folder there is test.fls.

And the info in my terminal says

```
Rc files read:
  NONE
Latexmk: This is Latexmk, John Collins, 29 May 2021, version: 4.74b.
Latexmk: making output directory './out'
Latexmk: making auxiliary directory './temp'
Latexmk: applying rule 'latex'...
Rule 'latex': File changes, etc:
   Changed files, or newly in use since previous run(s):
      'test.tex'
------------
Run number 1 of rule 'latex'
------------
------------
Running 'latex  -recorder -output-directory="out" -aux-directory="temp"  "test.tex"'
------------
This is pdfTeX, Version 3.141592653-2.6-1.40.22 (MiKTeX 21.6)
entering extended mode
(test.tex
LaTeX2e <2021-06-01> patch level 1
L3 programming layer <2021-08-27>
(/home/m4rx/.miktex/texmfs/install/tex/latex/base/article.cls
Document Class: article 2021/02/12 v1.4n Standard LaTeX document class
(/home/m4rx/.miktex/texmfs/install/tex/latex/base/size10.clo))
(/home/m4rx/.miktex/texmfs/install/tex/latex/amsmath/amsmath.sty
For additional information on amsmath, use the `?' option.
(/home/m4rx/.miktex/texmfs/install/tex/latex/amsmath/amstext.sty
(/home/m4rx/.miktex/texmfs/install/tex/latex/amsmath/amsgen.sty))
(/home/m4rx/.miktex/texmfs/install/tex/latex/amsmath/amsbsy.sty)
(/home/m4rx/.miktex/texmfs/install/tex/latex/amsmath/amsopn.sty))
(/home/m4rx/.miktex/texmfs/install/tex/latex/l3backend/l3backend-dvips.def)
(/home/test/temp/test.aux) [1] (/home/test/temp/test.aux) )
Output written on /home/test/out/test.dvi (1 page, 264 bytes).
Transcript written on /home/test/temp/test.log.
Latexmk: Moving 'temp/test.fls' to 'out/test.fls'
Latexmk: Examining 'temp/test.log'
=== TeX engine is 'pdfTeX'
Latexmk: Log file says output to 'out/test.dvi'
Latexmk: All targets (out/test.dvi) are up-to-date
```

You can see clearly, latexmk move the test.fls from 'temp/test.fls' to 'out/test.fls' in this line:

> Latexmk: Moving 'temp/test.fls' to 'out/test.fls'

So maybe fls file is indeed be considered as an aux file, but eventually be moved to output folder.
